counterparty-core/counterpartycore/lib/api/api_server.py

```diff
@@ -384,19 +384,35 @@ def init_flask_app():
         # Get the last block index
         with APIDBConnectionPool().connection() as db:
             util.CURRENT_BLOCK_INDEX = ledger.last_db_index(db)
+        methods = ["OPTIONS", "GET"]
         # Add routes
-        app.add_url_rule("/v2/", view_func=handle_route, strict_slashes=False)
         app.add_url_rule(
-            "/v2/blueprint", view_func=handle_doc, methods=["GET"], strict_slashes=False
+            "/v2/",
+            view_func=handle_route,
+            methods=methods,
+            strict_slashes=False,
+            provide_automatic_options=False,
```

Comment:
what does this arg do?

Reply:
by default Flask manage alone OPTIONS request and doesn't pass them to the application.